ansible/devstack.yml

```
@@ -1,6 +1,7 @@
 ---
 
 - hosts: "{{ instance_name }}"
+  remote_user: cloudbuild
```

This prevents `instance setup` from doing its job, here's a suggestion

```suggestion
  remote_user: "{{ username }}"
```

To make this work, in 
- [scripts/instance.sh#L123](https://github.com/appsembler/sultan/blob/cloudbuild/scripts/instance.sh#L123)
- [scripts/instance.sh#L225](https://github.com/appsembler/sultan/blob/cloudbuild/scripts/instance.sh#L225)

we need to append the following variable `username=$USER_NAME`
> `USER_NAME` already exists in the configs and printed in the debug message

---

@thraxil We still need to add `username=$USER_NAME` to the Ansible variables in the files above.